### DISCUSSION TITLE
upy-motion

### DISCUSSION THREAD

**Discussion by OneMadGypsy** [OneMadGypsy/upy-motion#1] [Announcements]
Welcome!
Feel free to discus problems, features, and suggestions or share personal projects pertaining to the upy-motion. Thank you for your interest in my project.

If you would like to officially file a bug report or request a feature you can use the following templates:
[bug report](https://github.com/OneMadGypsy/upy-motion/blob/main/.github/ISSUE_TEMPLATE/bug_report.md)
[feature request](https://github.com/OneMadGypsy/upy-motion/blob/main/.github/ISSUE_TEMPLATE/feature_request.md)

**Comment by xetrics** [comment]
Really awesome project, best driver for the MPU6050 I've found. Thanks for your work!

**Reply by OneMadGypsy** [reply]
Thank you.
